recipes/vectorclass/all/test_package/CMakeLists.txt

```diff
@@ -7,3 +7,6 @@ conan_basic_setup()
 add_executable(${PROJECT_NAME} test_package.cpp)
 target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
 set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 17)
+if (VECTORCLASS_HEADER_ONLY)
```

Comment:
It could be passed by package_info instead. It's not official from upstream, but the upstream doesn't provide anything relate to building, so I'm think is okay introducing a define.

Reply:
Not sure it's a good idea (I'm not even sure that header_only option is a good idea).